orcestra_book/operation/flight_categories.yaml

```diff
@@ -38,6 +38,12 @@ categories:
   meteor:
     long_name: "RV Meteor coordination"
     tier: "b"
+  halo:
+    long_name: "HALO coordination"
+    tier: "b"
+  atr:
+    long_name: "ATR coordination"
+    tier: "b"
```

Comment:
If I remember correctly, the idea is to use the `curtain` category whenever two or more aircrafts coordinate. Otherwise two categories, e.g. `halo` and `atr`, would actually describe the same situation. 

But maybe @bjorn-stevens can chime in here

Reply:
Ah ok I didn't know about that. Even if one aircraft is flying in situ?

Reply:
That is a good question 🤷‍♂️ I guess that's a question for the flight planning team /cc @JuliaWindmiller1987

Reply:
Yes, I think the idea to use "curtain" was to unify the aircraft coordination under one tag...

Reply:
OK I see, I can do that, even though it doesn't really explain our flight pattern. Do you want me to close the pull request, change it locally and make a new pull request?

Reply:
ok pull request updated